eth/chains/base.py

```diff
@@ -320,7 +335,7 @@ class Chain(BaseChain):
     current block number.
     """
     logger = logging.getLogger("eth.chain.chain.Chain")
-    gas_estimator = None  # type: Callable
+    gas_estimator = None     # type: Callable[[BaseState, BaseTransaction], int]
```

Comment:
I noticed a couple of type comments come with an unnecessary number of white space before. That should be changed.